docs/products/metrics/howto/migrate-m3db-thanos.md

```diff
@@ -0,0 +1,121 @@
+  `<metric-name>` in Thanos. Consequently, after migration, you may encounter two names
+  for the same metric:
+
+  - `prometheus_<metric-name>{_source=m3db, labels...}` (for time series migrated from M3DB)
+  - `<metric-name>{labels...}` (for time series written to Thanos directly)
+
+  Adjust your queries accordingly. The built-in dashboards provided by Aiven
+  services through integrations do not require any manual changes.
+- **Version compatibility:** Ensure your M3DB version supports migration to
+  Thanos Metrics. You might need to upgrade your M3DB to a compatible version.
+
+## Prerequisites
+
+Before starting the migration from an Aiven for M3DB service:
+
+- Make a note of the Aiven project and Aiven for M3DB service name for migration in the
+  [Aiven Console](https://console.aiven.io/).
+
+The [Aiven Console](https://console.aiven.io/) migration tool automatically uses
+connection details like the hostname, port, and credentials linked to the selected
+Aiven for M3DB service.
+
+## Database migration steps
+
+1. Log in to the [Aiven Console](https://console.aiven.io/) and select the
+   Aiven for Metrics service for your M3DB database migration.
+1. Go to **Service settings** from the sidebar.
+1. Scroll to the **Service management** section, and click
+   <ConsoleLabel name="actions"/> > **Import database** to initiate the import process.
+1. Follow the wizard to guide you through the database migration process.
+
+### Step 1: Configure
+
+1. From the drop-down menu, select your project name.
+2. From the subsequent drop-down, select the Aiven for M3DB database you intend to migrate.
+3. Click **Get started** to proceed with the migration.
+
+### Step 2: Validation
+
+The [Aiven Console](https://console.aiven.io/) automatically attempts to validate
+the database configurations for the selected Aiven for M3DB service.
+Click **Run validation** to validate the connection.
+
+:::warning
+If a validation error occurs during migration, follow the on-screen instructions to f
+ix it. Rerun validation to ensure the database meets migration criteria. Note that the
+migration doesn't include service user accounts and commands in progress.
+:::
+
+### Step 3: Migration
+
+Once all the necessary checks have been completed successfully, you can proceed with
+the migration process.
+
+- Click **Start migration** to initiate the data migration process to Aiven for Metrics.
+
+### Step 4: Replication
+
+While the migration is in progress:
+
+- You can close the migration wizard by clicking **Close window** and return later
+  to monitor the progress. Check the service's **Overview** page to track the
+  migration progress.
+- To stop the migration, click **Stop migration**. The data already transferred to
+  Aiven for Metrics is preserved.
+
+:::important
+To prevent conflicts during replication:
+
+- Avoid creating or deleting databases on the source service during migration.
```

Comment:
Terminology is a bit off here. There are no databases in m3db, but there are namespaces. Better say to avoid creating, deleting, or modifying namespaces.

Reply:
Good point!